patches/@metamask+network-controller+22.1.0.patch

```
@@ -0,0 +1,30 @@
+diff --git a/node_modules/@metamask/network-controller/dist/NetworkController.cjs b/node_modules/@metamask/network-controller/dist/NetworkController.cjs
+index cc9793f..b94112a 100644
+--- a/node_modules/@metamask/network-controller/dist/NetworkController.cjs
++++ b/node_modules/@metamask/network-controller/dist/NetworkController.cjs
+@@ -525,7 +525,11 @@ class NetworkController extends base_controller_1.BaseController {
+         let networkChanged = false;
+         const listener = () => {
+             networkChanged = true;
+-            this.messagingSystem.unsubscribe('NetworkController:networkDidChange', listener);
++            try {
++                this.messagingSystem.unsubscribe('NetworkController:networkDidChange', listener);
++            } catch {
++                // The controller may have been destroyed, so don't worry about it
```

What error is expected here? An empty catch block with silently suppress any errors.

---

Sure, good point. I've added a check for the error we want to ignore here: [52f4433](https://github.com/MetaMask/metamask-mobile/pull/12765/commits/52f4433e779eb48ed3cf75bb8f2e95723cb03b61)